x/wasm/keeper/handler_plugin_encoders_test.go

```diff
@@ -559,24 +628,174 @@ func TestEncoding(t *testing.T) {
 			output: []sdk.Msg{
 				&govtypes.MsgVote{
 					ProposalId: 1,
-					Voter:      addr1.String(),
+					Voter:      myAddr.String(),
 					Option:     govtypes.OptionNoWithVeto,
 				},
 			},
 		},
+		"Gov vote: unset option": { // regression test, unset = yes
```

Comment:
This is a regression test to cover this edge case. It will fail if `yes` is not the first element (=`0`) anymore so that we get notified of a breaking change. This setup is not ideal. Instead the first element should be 'Undefined' or `Yes = iota +1` in Go so that this case can be identified and handled proper.

Reply:
Good, happy to change that in wasmvm

Reply:
Is this better? https://github.com/CosmWasm/wasmvm/pull/392